src/pipeline/extract-methods.ts

```diff
@@ -30,7 +30,7 @@ export class ExtractMethodsStep extends PipelineStep {
         methodStartIndexArray.forEach((methodStartIndex: number) => {
             const tempMethods: string[] = [];
             const tempModifiedMethods: string[] = [];
-            for (let index = methodStartIndex; context.codeSnippetStringArray[index] != ''; index++) {
+            for (let index = methodStartIndex; /\S/.test(context.codeSnippetStringArray[index]); index++) {
```

Comment:
What is this doing?

Reply:
just an improvement of checking for whitespaces instead only a single string